src/queries.test.tsx

```diff
@@ -13,8 +13,14 @@ import {
   useSubmissionList,
 } from "./queries";
 import { produce } from "immer";
-import { patchMetadataSubmissionError, server, delay } from "./mocks/server";
+import {
+  patchMetadataSubmissionError,
+  server,
+  delay,
+  acquireLockConflict,
```

Comment:
I see you have used the word `acquire` here. I added a comment to a line way above this one (in a different file) about using a word other than `get` in a function name. In that context—if the options were `acquire` and `get`—I'd prefer `acquire`.